### PR TITLE
Update pagination docs

### PR DESCRIPTION
**Pull request by KenitoInc**
- [x] Server-driven pagination
- [x] Client-driven pagination

### CODE REVIEW

File: Odata-docs/client/v7/pagination.md [hunk 1]
```diff
@@ -1,60 +1,71 @@
...
 author: mumbi-o
 ms.author: mowambug
 ms.date: 7/1/2019
 ms.topic: article
  
 ---
-# Server driven paging
+# Pagination
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
+Loading large datasets can be slow. Services often rely on pagination to load the data incrementally to improve the response times and the user experience. Paging can be server-driven or client-driven:
```
Comment (gathogojr) [inline Odata-docs/client/v7/pagination.md:13]:
```suggestion
Loading large datasets can be slow. Services often rely on pagination to load the data incrementally to improve the response times and the user experience. Paging can be server-driven or client-driven.
```

File: Odata-docs/client/v7/pagination.md [hunk 2]
```diff
@@ -1,60 +1,71 @@
...
 
-The OData Client for .NET deals with server-driven paging with the help of `DataServiceQueryContinuation` and `DataServiceQueryContinuation<T>`. They are classes that contain the next link of the partial set of items.
+## Server-driven paging
+In Server-driven paging, the server returns the first page of results. If total number of results is greater than the page size, the server returns the first page along with a nextlink that can be used to fetch the next page of results.
+
+The OData Client deals with server-driven paging with the help of [DataServiceQueryContinuation](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation) and [DataServiceQueryContinuation&lt;T&gt;](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation-1). They are classes that contain the next link of the partial set of items.
```
Comment (gathogojr) [inline Odata-docs/client/v7/pagination.md:18]:
```suggestion
The OData Client deals with server-driven paging with the help of [DataServiceQueryContinuation](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation) and [DataServiceQueryContinuation&lt;T&gt;](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation-1). These classes contain the next link of the partial set of items.
```

File: Odata-docs/client/v7/pagination.md [hunk 3]
```diff
@@ -1,60 +1,71 @@
...
 // Get the first page
-var response = await context.People.ExecuteAsync() as QueryOperationResponse<Person>;
+QueryOperationResponse<Person> response = await context.People.ExecuteAsync() as QueryOperationResponse<Person>;
+int pageCount = 0;
 
 do
 {
-    if(nextLink != null)
+    pageCount++;
+    Console.WriteLine($"Page {pageCount}");
```
Comment (gathogojr) [inline Odata-docs/client/v7/pagination.md:37]:
```suggestion
    Console.WriteLine($"Page {++pageCount}");
```
and delete line above this one

File: Odata-docs/client/v7/pagination.md [hunk 4]
```diff
@@ -160,3 +167,26 @@ Person Page 2:
                .............................................
                .............................................
 ```
+
+## Client-driven paging
+In client-driven paging, we request the server to return the specified number of results. There is no next link that is returned.
```
Comment (gathogojr) [inline Odata-docs/client/v7/pagination.md:172]:
```suggestion
In client-driven paging, we request the server to return the specified number of results. There is no nextLink that is returned.
```

File: Odata-docs/client/v7/pagination.md [hunk 5]
```diff
@@ -160,3 +167,26 @@ Person Page 2:
...
+
+The OData Client deals with client-driven paging using `$skip` and `$top` in the query options.
```
Comment (gathogojr) [inline Odata-docs/client/v7/pagination.md:174]:
```suggestion
The OData Client deals with client-driven paging using `$skip` and `$top` query options.
```

File: Odata-docs/client/v7/pagination.md [hunk 6]
```diff
@@ -74,43 +85,39 @@ try
...
-                    Console.WriteLine("\t\tTrips Page {0}:", ++innerPageCount);
+                    Console.WriteLine($"\t\tTrips Page {++innerPageCount}:");
                 }
-                foreach (Trip t in tripsResponse)
+                foreach (Trip trip in tripsResponse)
                 {
                     // Print out the trips.
-                    Console.WriteLine("\t\t\tTripID: {0} - Name: ${1}",
-                        t.TripId, t.Name);
+                    Console.WriteLine($"\t\t\tTripID: {trip.TripId} - Name: ${trip.Name}");
```
Comment (gathogojr) [inline Odata-docs/client/v7/pagination.md:120]:
```suggestion
                    Console.WriteLine($"\t\t\tTripID: {trip.TripId} - Name: {trip.Name}");
```

File: Odata-docs/client/v7/pagination.md [hunk 7]
```diff
@@ -74,43 +85,39 @@ try
...
-                Console.WriteLine("\t\tTrips Page {0}:", ++innerPageCount);
+                Console.WriteLine($"\t\tTrips Page {++innerPageCount}:");
             }
-            foreach (Trip t in p.Trips)
+            foreach (Trip trip in person.Trips)
             {
                 // Print out the trips in the first page.
-                Console.WriteLine("\t\t\tTripID: {0} - Name: ${1}",
-                    t.TripId, t.Name);
+                Console.WriteLine($"\t\t\tTripID: {trip.TripId} - Name: ${trip.Name}");
```
Comment (gathogojr) [inline Odata-docs/client/v7/pagination.md:104]:
```suggestion
                Console.WriteLine($"\t\t\tTripID: {trip.TripId} - Name: {trip.Name}");
```

**Review by habbes — APPROVED** [review]

File: Odata-docs/client/v7/pagination.md (hunk 1 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 2 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 4 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 5 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 7 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 6 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

**Review by gathogojr — APPROVED** [review]

**Review by ElizabethOkerio — APPROVED** [review]

File: Odata-docs/client/v7/pagination.md [hunk 8]
```diff
@@ -1,60 +1,70 @@
...
  
 ---
-# Server driven paging
+# Pagination
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
+Loading large datasets can be slow. Services often rely on pagination to load the data incrementally to improve the response times and the user experience. Paging can be server-driven or client-driven.
 
-The OData Client for .NET deals with server-driven paging with the help of `DataServiceQueryContinuation` and `DataServiceQueryContinuation<T>`. They are classes that contain the next link of the partial set of items.
+## Server-driven paging
+In Server-driven paging, the server returns the first page of results. If total number of results is greater than the page size, the server returns the first page along with a nextlink that can be used to fetch the next page of results.
```
Comment (odero) [inline Odata-docs/client/v7/pagination.md:16]:
```suggestion
In Server-driven paging, the server returns the first page of results. If total number of results is greater than the page size, the server returns the first page along with a `nextLink` that can be used to fetch the next page of results.
```

File: Odata-docs/client/v7/pagination.md [hunk 9]
```diff
@@ -1,60 +1,70 @@
...
+
+The OData Client deals with server-driven paging with the help of [DataServiceQueryContinuation](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation) and [DataServiceQueryContinuation&lt;T&gt;](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation-1). These classes contain the nextLink of the partial set of items.
```
Comment (odero) [inline Odata-docs/client/v7/pagination.md:18]:
```suggestion
The OData Client deals with server-driven paging with the help of [DataServiceQueryContinuation](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation) and [DataServiceQueryContinuation&lt;T&gt;](/dotnet/api/microsoft.odata.client.dataservicequerycontinuation-1). These classes contain the `nextLink` of the partial set of items.
```

File: Odata-docs/client/v7/pagination.md [hunk 10]
```diff
@@ -1,60 +1,70 @@
...
 
 }
 // Loop if there is a next link
 while ((nextLink = response.GetContinuation()) != null);
 ```
 
-## Nested Pagination
+### Nested Pagination
 There are instances where we need to load pages of entities as well as pages of their related entities.
 The example below returns related Trips for each Person entity from the data service. It uses a do...while loop to paginate through Person entities and a nested while loop to paginate through the related Trips.
```
Comment (odero) [inline Odata-docs/client/v7/pagination.md:55]:
```suggestion
The example below returns related `Trips` for each `Person` entity from the data service. It uses a do...while loop to paginate through `Person` entities and a nested while loop to paginate through the related `Trips`.
```

File: Odata-docs/client/v7/pagination.md [hunk 11]
```diff
@@ -160,3 +166,26 @@ Person Page 2:
                .............................................
                .............................................
 ```
+
+## Client-driven paging
+In client-driven paging, we request the server to return the specified number of results. There is no nextLink that is returned.
```
Comment (odero) [inline Odata-docs/client/v7/pagination.md:171]:
```suggestion
In client-driven paging, we request the server to return the specified number of results. There is no `nextLink` that is returned.
```

**Review by odero — CHANGES_REQUESTED** [review]

File: Odata-docs/client/v7/pagination.md (hunk 8 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 9 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 11 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

File: Odata-docs/client/v7/pagination.md (hunk 10 above)
Comment (KenitoInc) [reply Odata-docs/client/v7/pagination.md]: #Resolved

**Pull request merged**